backend/app/adapter/db/url.go

```diff
@@ -96,6 +96,21 @@ WHERE "%s"=$1;`,
 	return url, nil
 }
 
+// GetByAliases finds URLs for a list of aliases
+func (u URLSql) GetByAliases(aliases []string) ([]entity.URL, error) {
+	urls := make([]entity.URL, 0)
+
+	for _, alias := range aliases {
+		url, err := u.GetByAlias(alias)
```

Comment:
I think it is a potential bottleneck. Let's imagine we want to fetch 1000 aliases, so we will perform 1000 sequent requests. I think we should do next:

* We implement the ability to pass the list of aliases and perform an operation by one query. ( SQL IN Operator )
* `GetByAlias` should call `GetByAliases` with providing the list of a single element.

In that case, we will have only one place with SQL logic and `GetByAlias` will be a helper

Reply:
I agree on you with performing an operation by one query with SQL IN Operator. However, I feel that ```GetByAlias``` should stay the same, since it does not need SQL IN Operator, which has a high time complexity.

Reply:
I have made changes to ```GetByAlias```, please take a look when you get the chance, thank you!